docs/code/draft_and_commit.py

```diff
@@ -31,6 +31,12 @@
 # is_draft = True (True for draft, False for commit)
 draft_number = dataset_client.status.draft_number
 # draft_number = 1
+branch_name = dataset_client.status.branch_name
```

Comment:
what is the meaning of `multi-version` in commit message?

Reply:
commit message：

docs: update version control chapter based on newly added Branch section

Please then add the concrete modifications to the commit message body.